charts/frr-k8s/templates/controller.yaml

```diff
@@ -49,7 +49,7 @@ data:
     # Check /etc/pam.d/frr if you intend to use "vtysh"!
     #
     vtysh_enable=yes
-    zebra_options="  -A 127.0.0.1 -s 90000000"
+    zebra_options="  -A 127.0.0.1 -K 120 -s 90000000"
```

Comment:
the commit says also `--graceful_restart argument ` but I don't see it. 
Also, what's special about 120 seconds? Can we document it (as a comment and in the commit description)?
Also also, would it make sense to have it as a parameter?

Reply:
The `-K` is the short version arg for the `--graceful_restart`, I prefered the short, do you prefer to use the long or should improve the commit title?

About the 120seconds, the 120 matches the 120 seconds that graceful-timeout advertise to the other side (https://docs.frrouting.org/en/latest/bgp.html#clicmd-bgp-graceful-restart-restart-time-0-4095) 

I do not think it should be parameter because we already hardcode the https://docs.frrouting.org/en/latest/bgp.html#clicmd-bgp-graceful-restart-restart-time-0-4095 and i do not see any benefit.